types/awsregion.pp

```diff
@@ -6,5 +6,5 @@
                             'eu-central-1',
                             'eu-west-1',
                             'sa-east-1',
-                            'us-gov-west-1',
+                            'us-gov-west-1'
```

Comment:
```
types/awsregion.pp:9:trailing_comma:WARNING:missing trailing comma after last element
```

Reply:
@igalic I think we may need to ignore the warnings until this is fixed in 5.2:
https://tickets.puppetlabs.com/browse/PUP-7952
Basically, the trailing slash is required by the linter, but fails on 5.2

IIRC, rubocop only flags the trailing comma on ruby hashes and puppet-lint only on .pp files, but I do always find it kind of funny / silly.